ranger/ext/human_readable.py

```diff
@@ -19,42 +18,61 @@ def human_readable(byte, separator=' '):  # pylint: disable=too-many-return-stat
     '1023 M'
     """
 
+    units = ['B', 'K', 'M', 'G', 'T', 'P']
+
     # handle automatically_count_files false
     if byte is None:
         return ''
 
-    if SettingsAware.settings.size_in_bytes:
-        return format(byte, 'n')  # 'n' = locale-aware separator.
+    # if SettingsAware.settings.size_in_bytes:
```

Comment:
Looks like you no longer take this setting into account?

Reply:
This made the module easily testable. This is part of the first commit which I didn't intended to be merged. I you want to keep the performance tests of the first commit in the repo, then I could figure out a way around this.

Reply:
I gave it about half an hour and I cannot figure out hot to make my performance tests work with SettingsAware. So I will simply drop the first commit. I don't think my performance tests are that useful and they have served their purpose. I could just place them to the suggested directory without being actually executable without this modification (commenting out SettingsAware), but I don't like that.